runtime/oti/cfr.h

```diff
@@ -902,6 +919,7 @@ typedef struct J9CfrClassFile {
 #define CFR_DECODE_J9_MULTIANEWARRAY	23
 #define CFR_DECODE_J9_METHODTYPEREF		24
 #define CFR_J9FLAG_HAS_JSR  		1
+#define CFR_J9FLAG_IS_RECORD        2
```

Comment:
Can you point at why cfdump needs this?  It would be cleaner to add this change (and related code) to the PR that requires it.

Reply:
I can move this to the cfdumper pr.
I use this in cfdumper as a way to identify a record class when displaying a J9CfrClassFile and not a loaded rom class.